<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game</class>
 <widget class="QWidget" name="game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="my_pokemon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="opponent_pokemon">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>301</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="exp_fight">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>升级赛</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pokemon_fight">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>115</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>决斗赛</string>
   </property>
  </widget>
  <widget class="QLabel" name="pro_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>属性  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="my_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>my_icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="my_pro">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>pro</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>类型  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="my_type">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>type</string>
   </property>
  </widget>
  <widget class="QLabel" name="att_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>攻击力：</string>
   </property>
  </widget>
  <widget class="QLabel" name="my_att">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>att</string>
   </property>
  </widget>
  <widget class="QLabel" name="def_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>防御力：</string>
   </property>
  </widget>
  <widget class="QLabel" name="spe_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>速度  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="HP_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>HP    :</string>
   </property>
  </widget>
  <widget class="QLabel" name="exp_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>等级  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="my_def">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>def</string>
   </property>
  </widget>
  <widget class="QLabel" name="my_spe">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>spe</string>
   </property>
  </widget>
  <widget class="QLabel" name="my_hp">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>hp</string>
   </property>
  </widget>
  <widget class="QLabel" name="my_level">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>level</string>
   </property>
  </widget>
  <widget class="QLabel" name="my_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>my_name</string>
   </property>
  </widget>
  <widget class="QLabel" name="opp_icon">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>opp_icon</string>
   </property>
  </widget>
  <widget class="QLabel" name="opp_name">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>opponent_name</string>
   </property>
  </widget>
  <widget class="QLabel" name="def_label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>370</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>防御力：</string>
   </property>
  </widget>
  <widget class="QLabel" name="spe_label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>速度  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="exp_label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>450</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>等级  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="opp_def">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>370</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>def</string>
   </property>
  </widget>
  <widget class="QLabel" name="opp_hp">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>hp</string>
   </property>
  </widget>
  <widget class="QLabel" name="opp_pro">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>pro</string>
   </property>
  </widget>
  <widget class="QLabel" name="opp_att">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>att</string>
   </property>
  </widget>
  <widget class="QLabel" name="opp_spe">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>spe</string>
   </property>
  </widget>
  <widget class="QLabel" name="opp_type">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>type</string>
   </property>
  </widget>
  <widget class="QLabel" name="HP_label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>HP    :</string>
   </property>
  </widget>
  <widget class="QLabel" name="pro_label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>330</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>属性  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="att_label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>攻击力：</string>
   </property>
  </widget>
  <widget class="QLabel" name="opp_level">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>450</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>level</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>330</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>类型  ：</string>
   </property>
  </widget>
  <widget class="my_Label" name="fight">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>my_Label</class>
   <extends>QLabel</extends>
   <header location="global">my_Label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
